frontend/src/components/user/UserCreateModal.vue

```diff
@@ -86,12 +86,20 @@ async function searchIdirUsers() {
   }
 }
 
-onMounted(() => {
-  // TODO: Map rbac groups to radio list to get cleaner labels
-  selectableGroups.value = [GroupName.NAVIGATOR, GroupName.NAVIGATOR_READ_ONLY];
+onMounted(async () => {
+  const yarsGroups: Array<Group> = (await yarsService.getGroups()).data;
+
+  const allowedGroups: Array<GroupName> = [GroupName.NAVIGATOR, GroupName.NAVIGATOR_READ_ONLY];
```

Comment:
`allowedGroups` keep being defined ad-hoc in multiple files, usually with the same `[GroupName.NAVIGATOR, GroupName.NAVIGATOR_READ_ONLY]` values and modified later. Sometimes even unshifted with the same values. Could we declare this `allowedGroups` value somewhere else to be imported for consistency? Some care needs to be taken that the array imported isn't by reference when later modified with unshift.

Reply:
Gonna keep it as is, talked with Kyle and he agrees. Maybe could make utils function to handle this but this will be the only places this is used.